sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -6,9 +6,6 @@
     <AssemblyTitle>Microsoft Azure.Storage.Blobs client library</AssemblyTitle>
     <DefineConstants>BlobSDK;$(DefineConstants)</DefineConstants>
     <PackageTags>Microsoft Azure Storage Blobs, Microsoft, Azure, Blobs, Blob, Storage, StorageScalable, azureofficial</PackageTags>
-    <PackageReleaseNotes>
```

Comment:
This doesn't have a corresponding addition of changelog because the file already exists?